### ISSUE TITLE
plugin/log - ability to log denial AND error DNS responses

### ISSUE THREAD

**Issue by fturib** [coredns/coredns#1662]
For monitoring purpose we would need the ability to log ont only one class, but several.
typically : Denial + Error

stanza would looks like:

```
. {
    log example.org {
        class denial error
    }
}
```

in another word, "class" parameter would be an array instead of a single value.

**PR is almost ready - WIP**

**Comment by mparamonau** [comment]
PR is [here](https://github.com/coredns/coredns/pull/1664)

**Comment by miekg** [comment]
closed via #1664
